canonical-webteam/deployment-configs
# Configs for discourse-docs.snapcraft.io

QA
--

``` bash
echo "127.0.0.1 discourse-docs.snapcraft.io" | sudo tee -a /etc/hosts
./qa-deploy --production discourse-docs.snapcraft.io --tag release-candidate
```

Now go to http://discourse-docs.snapcraft.io and see the discourse-based docs site in all its glory.
